download/download.go

```diff
@@ -123,7 +125,7 @@ func (d *Downloader) loop() {
 			delay = util.DefaultBackoff(float64(minRetryDelay), float64(*d.config.Polling.MaxDelaySeconds), retry)
 		}
 
-		d.logDebug("Waiting %v before next download/retry.", delay)
+		d.client.Logger().Debug("Waiting %v before next download/retry.", delay)
```

Comment:
Why are we using `d.client.Logger()` and not `d.logger.Debug` ?

Reply:
Thanks for spotting, that was just by mistake. Fixed!